src/main/java/ticTacToe/grid/GridFactory.java

```diff
@@ -0,0 +1,8 @@
+package ticTacToe.grid;
+
+public class GridFactory {
+
+    public Grid createGrid(int userInput) {
```

Comment:
Is this going to contain more logic in upcoming PRs? Just wondering as the factory right now doesn't  add much other than that clients can call `createGrid` instead of `new Grid`.